test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
```

Comment:
As discussed: Exclude `tests` for all the tooling for now. We can always re-add them, but it's not a battle we want to be fighting right now I think.